src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/util/LoggingUtility.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.reform.wacaseeventhandler.util;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import uk.gov.hmcts.reform.wacaseeventhandler.exceptions.LoggingUtilityFailure;
+
+public final class LoggingUtility {
+
+    private static final ObjectMapper MAPPER = new ObjectMapper()
+        .setPropertyNamingStrategy(PropertyNamingStrategies.LOWER_CAMEL_CASE)
+        .registerModule(new JavaTimeModule())
+        .registerModule(new Jdk8Module());
```

Comment:
Could this be put into Spring configuration and injected here?

Reply:
will do that. i think it make sense and will avoid code duplication.